src/Pages/JobDetails/Components/Attachment.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const Attachment = ({
+  id,
+  fullName,
+  date,
+  status,
+  setSubmitReady,
+  submitReady,
+}) => {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleClick = e => {
+    setIsClicked(!isClicked);
+
+    // if (isClicked === true) {
+    //   setSubmitReady(true);
+    // }
+  };
+
+  useEffect(() => {
+    isClicked ? setSubmitReady(true) : setSubmitReady(false);
```

Comment:
- 
```suggestion
setSubmitReady(isClicked);
```

- 위 리뷰 반영해서 isClicked 스테이트를 지운다면, useEffect 부분도 필요 없어지겠네요!

Reply:
성훈님 요부분 질문 있습니다... 체크박스를 클릭하면 isClicked 가 true 로 바뀐 후 submitReady도 true 가 되어야하는데 useEffect를 안쓰면 isClicked의 업데이트된 state 가 반영이 안됩니다 ㅠㅠ 일단 useEffect 안에서 `setSubmitReady(isClicked);` 로 수정하였습니다!